package-lock.json

```diff
@@ -17063,6 +17055,14 @@
         "websocket-driver": "0.6.5"
       },
       "dependencies": {
+        "faye-websocket": {
+          "version": "0.10.0",
+          "resolved": "https://registry.npmjs.org/faye-websocket/-/faye-websocket-0.10.0.tgz",
+          "integrity": "sha1-TkkvjQTftviQA1B/btvy1QHnxvQ=",
+          "requires": {
+            "websocket-driver": ">=0.5.1"
+          }
+        },
         "websocket-driver": {
```

Comment:
Just trying to understand the fix, so rolling back to version 0.10.0 fixes the issue, right? Cool. Should this also be listed in the `package.json` file? I probably missed it but I couldn't find any mention of a dependency for `faye-websocket`.